File: lib_common/src/main/java/com/yunzhu/module/common/common/RxEvent.java
package com.yunzhu.module.common.common;

public class RxEvent {

    public int what;
    public Object event;

    public RxEvent() { }

    /**
     * RxBus 事件
     * @param what       事件类型
     * @param event      事件对象
     */
    public RxEvent(int what, Object event) {
        this.what = what;
        this.event = event;
    }

    public Object getEvent() {
        return event;
    }

}

File: lib_common/src/main/java/com/yunzhu/module/common/common/RxEvent.kt
package com.yunzhu.module.common.common

class RxEvent {
    var what = 0
    var event: Any? = null

    constructor() {}

    /**
     * RxBus 事件
     * @param what       事件类型
     * @param event      事件对象
     */
    constructor(what: Int, event: Any?) {
        this.what = what
        this.event = event
    }
}